import_and_export_data/guides/create-connector.rst

```diff
@@ -315,3 +315,66 @@ If you don't see your changes, make sure you have run (``bin/console assets:inst
 Now that we have our extension loaded in our form, we can add some logic into it, check how to `customize the UI`_.
 
 .. _customize the UI: ../../design_pim
+
+Make a job stoppable
+--------------------
+
+As of now, it is possible your jobs won't be able to stop when the job is started.
```

Comment:
Isn't weird to start by saying it does not work sometimes?

Reply:
because it depends on their custom job. If they use item steps it might work, if they use something else, it's possible it does not :/

Reply:
What do you think about:

In Akeneo 5.0 we added the possibility to declare a Job as stoppable. But as some task may be not stoppable (important workflows that always need to finish, job depending on other steps, etc) we decided to make this new feature "opt in". In this documentation, we will explain you how to make your job stoppable.

Reply:
fixed

Reply:
I agree with Julien, it's better like this.